diyspk/imapfilter/src/service-setup.sh

```diff
@@ -0,0 +1,11 @@
+service_postinst ()
+{
+    ln -s ${SYNOPKG_PKGDEST}/bin/imapfilter /usr/local/bin/imapfilter
+    ln -s ${SYNOPKG_PKGDEST}/share/imapfilter /usr/local/share/
```

Comment:
I would not link that folder. I propose to build tool replacing default "share" location to `/var/packages/$(PKG_NAME)/target/share/` - with variable to work either with diyspk/imapfilter or spk/synocli-net